pkg/gofr/datasource/mongo/mongo_test.go

```diff
@@ -489,8 +490,9 @@ func Test_HealthCheck(t *testing.T) {
 		cl.Database = mt.DB
 		mt.AddMockResponses(mtest.CreateSuccessResponse())
 
-		resp := cl.HealthCheck()
+		resp, err := cl.HealthCheck()
 
+		assert.Nil(t, err)
```

Comment:
```suggestion
		assert.NoError(t, err)
```

Reply:
I will open a PR to enable testifylint on updated code (not whole codebase)

https://github.com/gofr-dev/gofr/issues/787#issuecomment-2209629728